slither/core/scope/scope.py

```diff
@@ -95,7 +95,9 @@ def add_accesible_scopes(self) -> bool:
 
     def get_contract_from_name(self, name: Union[str, Constant]) -> Optional[Contract]:
         if isinstance(name, Constant):
-            return self.contracts.get(name.name, None)
+            name = name.name
+        if name in self.renaming:
+            name = self.renaming[name]
         return self.contracts.get(name, None)
```

Comment:
I think we should only fallback to the renaming if the contract name is not found to minimize potential collisions.